site/components/policy/policy-box/index.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+
+import React from 'react';
+
+import Statements from '../statements';
+import { H2 } from '../../text';
+
+import styles from './style.css';
+
+type StatementT = {
+  body: Function,
+};
```

Comment:
Can the Statement type also be imported from 'statements.js'  rather than being defined twice?